### ISSUE TITLE
README.md

### ISSUE THREAD

**Issue by kokitsuyuzaki** [rikenbit/dcTensor#5]
@dekuenstle 's comment

> The only installation instructions on the repository readme are aimed more at developers than end users because they install from Github instead of CRAN.

> The readme could be a good place to give users and developers separate instructions and add the missing community guidelines.

@CeciliaCoelho 's comment

> The repository readme is very poor.

> It should have a brief explanation at the top stating what the repository has.

> It can be the summary used in the paper after fixing according to 2.

> It should clearly state how to use the functions provided and how to run the example code.

> The readme should also include the contributing.md.

> All information should be readily available to the users.

> Again I suggest the authors to check readme files of accepted papers if they need a guideline.

**Comment by kokitsuyuzaki** [comment]
@dekuenstle 's comment

> The readme could be a good place to give users and developers separate instructions and add the missing community guidelines.

I separated the instructions for users and developers and added the missing community guidedlines in README.md.
https://github.com/rikenbit/dcTensor/commit/d00293df9b86e2f7484f04ec822d22bb699f7bd4#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

**Comment by kokitsuyuzaki** [comment]
@CeciliaCoelho 's comment

> It should have a brief explanation at the top stating what the repository has.

I added a brief explanation.
https://github.com/rikenbit/dcTensor/commit/d00293df9b86e2f7484f04ec822d22bb699f7bd4#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

> It can be the summary used in the paper after fixing according to 2.

> It should clearly state how to use the functions provided and how to run the example code.

> All information should be readily available to the users.

> Again I suggest the authors to check readme files of accepted papers if they need a guideline.

In my opinion, a lengthy README discourages users from reading, so it is not a good idea to put all the information here. I have written the usage of each function in a separate description, so I added the links to such descriptions.
https://github.com/rikenbit/dcTensor/commit/d00293df9b86e2f7484f04ec822d22bb699f7bd4#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

> The readme should also include the contributing.md.

I added the missing community guidedlines in README.md.
https://github.com/rikenbit/dcTensor/commit/d00293df9b86e2f7484f04ec822d22bb699f7bd4#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5
